pkg/utils/runtime_condition.go

```diff
@@ -56,8 +56,9 @@ func UpdateRuntimeCondition(conditions []data.RuntimeCondition, condition data.R
 	return conditions
 }
 
-// get runtime condition given a runtime condition type.
-// If found, return index of the founded condition in the condition array and the founded condition itself, otherwise return -1 and nil.
+// GetRuntimeCondition is given a runtime condition type.
```

Comment:
I suggest changes like:
```suggestion
// GetRuntimeCondition gets runtime condition given a runtime condition type.
```